_inc/lib/class.jetpack-google-maps-api-key.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+class Jetpack_Google_Maps_Api_Key {
+	static $option_key = 'jetpack_google_maps_api_data';
+
+	// return back the api key
+	function get( $key ) {
```

Comment:
Since you are returning `$key` on line 9 if `$key` is not empty, and not setting `$key` anywhere else, the `return $key` on line 22 is a bit confusing since it's either false or an empty string (or else we would have returned from this function on line 9).

I think it would be clearer if you refactored the function like this, replacing the `false` returned when `false === $api_key` with an empty string if necessary:

```
<?php

function get( $key ) {
	if ( ! empty( $key ) ) {
		return $key;
	}

	$api_key = get_option( self::$option_key, false );

	if ( false === $api_key ) {
		$this->maybe_refresh_on_shutdown();
		return false;
	}

	if ( isset( $api_key->expires ) && $api_key->expires < time() ) {
		$this->maybe_refresh_on_shutdown();
	}

	return $api_key->key;
}
```